tasks/main.yml

```diff
@@ -6,5 +6,8 @@
 - include: tools.yml
   when: galaxy_tools_install_tools
 
+- include: workflows.yml
+  when: galaxy_tools_workflows|length > 0
```

Comment:
@afgane let me know if you prefer an explicit `galaxy_tools_install_workflow` variable.